gamestonk_terminal/main_helper.py

```diff
@@ -525,36 +560,38 @@ def quote(other_args: List[str], s_ticker: str):
 
 
 def view(other_args: List[str], s_ticker: str, s_interval, df_stock):
-    """
-    Plot loaded ticker or load ticker and plot
+    """Plot loaded ticker or load ticker and plot
+
     Parameters
     ----------
```

Comment:
I actually changed this recently to just be plotting so it should be """Plot loaded ticker